contracts/src/CrossChainApplications/GETTING_STARTED.md

```diff
@@ -277,44 +309,59 @@ To start, replace the import for `ITeleporterReceiver` with `TeleporterOwnerUpgr
 Also, replace the contract declaration to inherit from `TeleporterOwnerUpgradeable` instead of `ITeleporterReceiver`:
 
 ```diff
-contract ExampleCrossChainMessenger is
+contract MyExampleCrossChainMessenger is
      ReentrancyGuard,
 -    ITeleporterReceiver
 +    TeleporterOwnerUpgradeable
-{}
+{
 ```
 
 Next, update the constructor to invoke the `TeleporterOwnerUpgradeable` constructor.
 
 ```diff
-- constructor(address teleporterMessengerAddress) {
--     teleporterMessenger = ITeleporterMessenger(teleporterMessengerAddress);
-- }
-+ constructor(
-+     address teleporterRegistryAddress,
-+     address teleporterManager
-+ ) TeleporterOwnerUpgradeable(teleporterRegistryAddress, teleporterManager) {}
+-     constructor(address teleporterMessengerAddress) {
+-         teleporterMessenger = ITeleporterMessenger(teleporterMessengerAddress);
+-     }
++     constructor(
++         address teleporterRegistryAddress,
++         address teleporterManager
++     ) TeleporterOwnerUpgradeable(teleporterRegistryAddress, teleporterManager) {}
 ```
 
-Then, remove the `teleporterMessenger` state variable, and add a call to get the latest `ITeleporterMessenger` implementation from `TeleporterRegistry` in `sendMessage`.
+Then, remove the `teleporterMessenger` state variable:
 
 ```diff
-- ITeleporterMessenger public immutable teleporterMessenger;
+-     ITeleporterMessenger public immutable teleporterMessenger;
+```
+
+And at the beginning of `sendMessage()` add a call to get the latest `ITeleporterMessenger` implementation from `TeleporterRegistry`:
+
+```solidity
+    function sendMessage(
+        ...
+    ) external returns (bytes32 messageID) {
+```
+```solidity
```

Comment:
Code blocks appear off

Reply:
Do we want to remove this triple code block as well?

Reply:
Ah yes indeed, thank you, done in https://github.com/ava-labs/teleporter/commit/3814571134a29106bf2d0ccb57fc149efd24a370